pom.xml

```diff
@@ -22,12 +22,12 @@
   <parent>
     <groupId>org.jboss</groupId>
     <artifactId>jboss-parent</artifactId>
-    <version>19</version>
+    <version>36</version>
   </parent>
 
-  <groupId>org.jboss.maven.plugins</groupId>
+  <groupId>com.github.dperezcabrera</groupId>
```

Comment:
Why does the groupId is changed from JBoss to personal one?

Reply:
I'm  so sorry, I just wanted to send the first commit as PR. 
Later I just did some other tests..